repos/system_upgrade/el7toel8/models/vsftpdfacts.py

```diff
@@ -0,0 +1,17 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+
+class VsftpdConfig(Model):
+    topic = SystemInfoTopic
+
+    path = fields.String()
+    strict_ssl_read_eof = fields.Nullable(fields.Boolean())
+    tcp_wrappers = fields.Nullable(fields.Boolean())
```

Comment:
If `tcp_wrappers is None`, it means that the "tcp_wrappers" option was left unspecified in the configuration file, so vsftpd will use the default value for the option. We could assign the default value to the attribute, if the option is unspecified in the config file, however I think it's better the way I designed it - this way the config reader is more generic, because reading the config, and deciding what the default value is, is separated.

Reply:
Discussed in person. Nullable makes sense. will be documented.